roles/ec2-rds/tasks/start-rds-instances.yml

```diff
@@ -20,4 +20,18 @@
 
   when:
     - start_stack|default(False)
-    - rds_instance_facts.stdout|search('stopped')
\ No newline at end of file
+    - rds_instance_facts.stdout|search('stopped')
+
+
+- block:
+    - name: Wait for rds to have started before continuing
```

Comment:
If this wait will only run on `stopped` instances, can you change the naming to make this clearer?